.github/workflows/playwright.yml

```diff
@@ -0,0 +1,35 @@
+name: Playwright tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  NODE_JS: "18"
+
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: ${{ env.NODE_JS }}
+
+    - uses: ./.github/actions/npm
+
+    - name: Install Playwright Browsers
```

Comment:
You can get rid of this step and speed things up by using the playwright container. Ex:
```
    container:
      image: mcr.microsoft.com/playwright:v1.37.1-jammy
```

Reply:
Good spot, thanks!

Reply:
Good spot, thanks!